.github/workflows/pr-mirror-repo-sync.yml

```
@@ -0,0 +1,35 @@
+          - main
+
+permissions:
+  pull-requests: write
+  contents: write
+  issues: write
+
+jobs:
+  call-pr-mirror-repo-sync:
+    if: |
+      github.repository_owner == 'vyos' &&
+      (a
```

actionlint (1.7.12)

[error] unexpected token "IDENT" while parsing closing ')' of nested expression (...). expecting ")"

(expression)